README.md

```diff
@@ -54,79 +20,63 @@ After you're finished please remove all the comments and instructions!
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 Ruby-Enumerator <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+This program creates a class MyList and a module MyEnumerable.
+The module MyEnumerable implements a subset of the functionality of Enumerable.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://www.ruby-lang.org/">Ruby</a></li>
+    <li><a href="https://ubuntu.com/">Terminal</a></li>
+    <li><a href="https://code.visualstudio.com/">Visual Studio</a></li>
   </ul>
 </details>
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **In this project we will learn how to use a module inside class. For this we create a class MyList and a module MyEnumerable. our module MyEnumerable will implement a subset of the functionality of Enumerable.**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
+<!-- GETTING STARTED -->
 
-## 🚀 Live Demo <a name="live-demo"></a>
+## 💻 Getting Started <a name="getting-started"></a>
 
-> Add a link to your deployed project.
+To get a local copy up and running, follow these steps.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+### Setup
```

Comment:
- [ ] Kindly fix the getting started section of the readme file. This should be an example of how you may give instructions on setting up your project locally. Currently, it has a duplicate setup and installs headline and it will make users confused. Kindly remove the duplicate things and keep the right ones that are needed to set up and run the project.

Reply:
## Hi @mahbubul14 ,
I made all changes in the README file in this current branch [My_Enumerable], kindly double check-it